client/app/models/landuse-form.js

```diff
@@ -75,9 +82,6 @@ export default class LanduseFormModel extends Model {
 
   @attr dcpSitedataidentifylandmark;
 
-  // Environmental Review attrs
```

Comment:
Should leave this comment in the file. I placed these comments here to section off these attributes

Reply:
Whoops that was an accident!